src/elements/content-sidebar/AdditionalTab.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * @flow
+ * @file Preview sidebar additional tab component
+ * @author Box
+ */
+
+import * as React from 'react';
+import noop from 'lodash/noop';
+import Tooltip from '../common/Tooltip';
+import PlainButton from '../../components/plain-button/PlainButton';
+import './AdditionalTab.scss';
+
+const AdditionalTab = ({ callback: callbackFn = noop, iconUrl, id, title, ...rest }: AdditionalTab) => {
+    return (
+        <Tooltip position="middle-left" text={title}>
+            <PlainButton
+                className="bcs-additional-tab-btn bcs-nav-btn"
+                type="button"
+                onClick={() => callbackFn({ id, rest })}
```

Comment:
This looks like a typo, since `...rest` is usually re-spread in callbacks. You may want to rename this as `callbackData` or something to indicate it's intended to be passed-through as an object literal.